cluster/stone-prd-rh01/tenants/rh-trex-tenant/release-plan.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+  name: rh-trex-release-to-quay
+spec:
+  application: rh-trex
+  target: rh-trex-tenant
```

Comment:
The target needs to be the managed namespace. In this case: `rh-managed-mng-s-2-tenant`

Reply:
Great catch - should be patched!